lib/cli/src/frameworks/react/js/Button.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const baseStyles = {
```

Comment:
I think we discussed putting these styles in an imported CSS file and using class names to switch between them. That way the amount of code we share between frameworks could be larger whilst still writing components in a "natural" way. 

WDYT @shilman?

Reply:
I'm in favor of moving everything that is not tied to a framework in the common folder.